README.md

```diff
@@ -85,20 +91,22 @@ Info below subject to change
       - For running app on iOS or Android emulator
     - Note that arguments can be combined, e.g. `yarn test --android --tunnel`
 
-### Building for Production Mode (Prebuild)
+### Prebuild Instructions for Android (Production Mode)
 
-**Android**
+Presumably compatible with both MacOS and Windows PC for building the project in Android Studio.
 
 - `yarn prebuild`
 - `yarn android`
 - Steps (to be fleshed out later):
   - Open Android Studio and build APK
   - Load it onto your phone
 
-**iOS**
+### Prebuild Instructions for iOS (Production Mode, needs MacOS)
+
+Probably needs a MacOS at least for building the project in XCode.
 
 - `yarn prebuild`
-- `yarn android`
+- `yarn ios`
```

Comment:
I don't really see why one would run yarn ios over yarn i so prob don't include this step. yarn ios exist because yarn prebuild will automatically populate the command